doc/man/pyinstaller.rst

```diff
@@ -23,7 +23,7 @@ PyInstaller is a program that freezes (packages) Python programs into
 stand-alone executables, under Windows, GNU/Linux, Mac OS X,
 FreeBSD, OpenBSD, Solaris and AIX.
 Its main advantages over similar tools are that PyInstaller works with
-Python 3.5—3.7, it builds smaller executables thanks to transparent
+Python 3.6—3.7, it builds smaller executables thanks to transparent
```

Comment:
Not related to that PR, but this should be:
```suggestion
Python 3.6—3.9, it builds smaller executables thanks to transparent
```

Reply:
Ah thanks, somehow didn't spot this.